elements/lisk-p2p/src/p2p_types.ts

```diff
@@ -35,51 +37,87 @@ export interface P2PPenalty {
 	readonly penalty: number;
 }
 
+export interface P2PSharedState {
+	readonly height: number;
+	readonly updatedAt?: Date;
+	readonly os?: string;
+	readonly version: string;
+	readonly protocolVersion: string;
+	// tslint:disable-next-line: no-mixed-interface
+	readonly [key: string]: unknown;
+}
+
+export interface P2PInternalState {
+	readonly dateAdded?: Date;
+	readonly peerKind?: PeerKind;
+	readonly isBanned?: boolean;
```

Comment:
This is not used.

Reply:
Right now only connectionKind value is being used, the rest of the values in internalState will be handled in #4314 issue, including if a peer is banned or not